src/crop_image.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
```

Comment:
Add a comment header, see https://github.com/julianweisbord/Deep-Learning-For-Object-Recognition-on-a-Mobile-Robot/blob/master/src/learning/model.py